fix regex that doesn't handle directory names with a double dash (--) in them correctly
this fix is specific to using '-32bit' as versionsuffix
author: Kenneth Hoste (HPC-UGent)
--- l_mkl_p_10.2.6.038/pset/install.sh.orig	2016-03-04 15:28:03.092055076 +0100
+++ l_mkl_p_10.2.6.038/pset/install.sh	2016-03-04 15:27:44.071603046 +0100
@@ -1281,7 +1281,7 @@
 	
 	[ $err -eq ${ERR_OK} ] || return 1
 		
-        RS=$(echo $CMD_STR | sed s/.*--$cmd[[:blank:]]*//g | sed 's/[[:blank:]]*--.*$//g')
+        RS=$(echo $CMD_STR | sed s/.*--$cmd[[:blank:]]*//g | sed 's/[[:blank:]]*--[^3].*$//g')
         [[ -z "$RS" ]] && return 1
         echo $RS
 	
